lib/ui/process_ui/widgets/language_selector.dart

```diff
@@ -78,197 +89,300 @@ class _LanguageSelectorState extends State<LanguageSelector> {
         .getAudit("REG-LOAD-006", "REG-MOD-103");
   }
 
+  _getNotificationLabel() {
+    String notificationLanguage = "";
+    context.watch<GlobalProvider>().chosenLang.forEach((element) {
+      String code = context.read<GlobalProvider>().languageToCodeMapper[element]!;
+      notificationLanguage += " / ${AppLocalizations.of(context)!.notificationLanguage(code)}";
+    });
+    return notificationLanguage.substring(3);
+  }
+
   @override
   Widget build(BuildContext context) {
     int minLanguage = context.read<GlobalProvider>().minLanguageCount;
     int maxLanguage = context.read<GlobalProvider>().maxLanguageCount;
-    final size = MediaQuery.of(context).size;
+    isMobile = MediaQuery.of(context).orientation == Orientation.portrait;
     return AlertDialog(
-      title: const Text(
-        "Select Language",
-        style: TextStyle(fontSize: 18, fontWeight: FontWeight.bold),
+      title: Text(
+        AppLocalizations.of(context)!.select_language,
+        style: TextStyle(
+            fontSize: isMobile ? 24 : 18, fontWeight: FontWeight.bold),
       ),
-      contentPadding: const EdgeInsets.all(4),
+      contentPadding: EdgeInsets.symmetric(horizontal: 20.w, vertical: 20.h),
       content: SizedBox(
-        height: size.width < 512 ? size.height / 2 : size.height,
+        width: isMobile ? 644.w : 824.32.w,
         child: Column(
           crossAxisAlignment: CrossAxisAlignment.start,
+          mainAxisSize: MainAxisSize.min,
           children: [
             const Divider(),
             ListTile(
               minLeadingWidth: 0,
               title: Text(
-                "Please select any $minLanguage language for data entry",
-                style: Theme.of(context).textTheme.bodyMedium,
+                AppLocalizations.of(context)!.select_two_languages,
+                style: const TextStyle(
+                  fontSize: 22,
+                ),
               ),
               leading: const Icon(
                 Icons.check,
                 color: Colors.green,
               ),
             ),
-            ListTile(
+             ListTile(
               minLeadingWidth: 0,
               title: Text(
-                "English is Mandatory for the residents to select as per government guidelines. The order of selection will lead to page view for demographic data.",
-                style: Theme.of(context).textTheme.bodyMedium,
+                AppLocalizations.of(context)!.english_mandatory,
```

Comment:
can we rename it to language_mandatory.
how will this be affected if there isn't any language which is mandatory as per the spec